apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -142,17 +143,33 @@ model GuildContributionVerificationStatus {
 }
 
 model GuildActivityType {
-  id               Int          @id @default(autoincrement())
-  createdAt        DateTime     @default(now())
-  updatedAt        DateTime     @default(now()) @updatedAt
-  guild_id         Int
-  guild            Guild        @relation(fields: [guild_id], references: [id])
-  activity_type_id Int
-  activity_type    ActivityType @relation(fields: [activity_type_id], references: [id])
+  id                         Int                          @id @default(autoincrement())
+  createdAt                  DateTime                     @default(now())
+  updatedAt                  DateTime                     @default(now()) @updatedAt
+  guild_id                   Int
+  guild                      Guild                        @relation(fields: [guild_id], references: [id])
+  activity_type_id           Int
+  activity_type              ActivityType                 @relation(name: "guilds", fields: [activity_type_id], references: [id])
```

Comment:
the custom name guilds doesn't seem to accurately describe the relation between the columns, as it is a foreign key to activity types. Please check my other comments.

Reply:
In this case, using descriptive name that reflects the relation between two models ensures readability for constraints names. For examples, using a more specific name than "guilds" would definitely help to avoid confusion.